### PR TITLE
Update Elasticsearch recommended version to 7.6

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [x] Documentation Update

## Description

As we're now safe and sound both in both client and server on ES 7.6, I've updated the recommended version, removed the hack in macOS's instructions and recommended Homebrew and updated the version in Docker and Travis.

## Related Tickets & Documents

#7208 

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [x] docs.dev.to
- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

File: docs/installation/macos.md [hunk 1]
```diff
@@ -60,33 +60,52 @@ redis-cli ping
 
 ### Elasticsearch
 
-DEV requires a version of Elasticsearch between 7.1 and 7.5. Version 7.6 is not
-supported. We recommend version 7.5.2.
+DEV requires Elasticsearch 7+. We recommend version 7.6.
 
 You have the option of installing Elasticsearch with Homebrew or through an
-archive. We recommend installing from archive on Mac.
+archive. We recommend installing from Homebrew as it will be kept up to date
```
Comment (Ridhwana) [inline docs/installation/macos.md:66]: 🎉whoops whoop for this

**Comment by rhymes** [comment]
@mstruve oooh! I thought we had 7.6 in production, that's why I sent this PR :-| Sorry I must have misunderstood

**Comment by mstruve** [comment]
Nope, still 7.5.2 in production. Seeing that 7.6 cleans some things up I put in a card to upgrade us. https://github.com/thepracticaldev/dev.to/projects/6#card-36275050

**Comment by rhymes** [comment]
@mstruve cool! I'll dig this out when we have upgraded and safely tested!

**Pull request closed without merging**